README.md

```diff
@@ -315,16 +327,14 @@ Your transaction should then look similar to this example of a BookmarksControll
 
 ```php
 $trail = new AuditTrail();
-$success = $this->Bookmarks->connection()->transactional(function () use ($trail) {
+$success = $this->Bookmarks->connection()->transactional(function () use ($trail): void {
```

Comment:
i think something can be returned - why this change ?

Reply:
According to cakephp docs the closure do not need to return anything to make it work, but for clarity i can remove `void` as return type from this example